src/Resources/assets/controllers/dropdown_controller.js

```diff
@@ -40,11 +43,22 @@ export default class extends Controller {
         dropdownDiv.style.left = '';
 
         // Calculate
-        const {top, left} = dropdownDiv.getBoundingClientRect();
+        const {top, left, bottom, height} = dropdownDiv.getBoundingClientRect();
+        const windowHeight = document.body.clientHeight;
 
         // Set fixed position
         dropdownDiv.style.position = 'fixed';
-        dropdownDiv.style.top = top + 'px';
         dropdownDiv.style.left = left + 'px';
+        if(this.alignmentValue === 'bottom') {
+            let topPosition = top;
+            if(bottom > windowHeight) {
+                topPosition = windowHeight - height;
+            }
+            dropdownDiv.style.top = topPosition + 'px';
```

Comment:
Nit: Use template literals where possible. It's just the more appropriate and readable way of string chaining in JavaScript nowadays. 

e.g. `${topPosition}px`.